README.md

```diff
@@ -71,6 +71,18 @@ Using the `traveler` object, update the DOM to show the traveler's *name*, *nick
 
 **YOUR NOTES**
 ```
+I'm not sure why we have access to the traveler variable declared in data.js, but I think it might be because index.html calls both js files in script tags.
+
```

Comment:
Correct, however remember that HTML is loaded top down so the order of the script tags matter. Switching the script tag order will make it so that 'index.js' does not have access to 'data.js' since 'index.js' will be loaded before 'data.js':

 <script src="index.js"></script>
 <script src="data.js"></script>